### PR TITLE
[MTSRE-448] Integrating custom output table and adding version information for addons

### PR DESCRIPTION
**Pull request by Ajpantuso**
### Summary

This PR overhauls table processing by removing tables built from the _ocm-cli_ `output` package. It also adds a sub request to add version information for addons with the `list` sub-command.

### CODE REVIEW

**Comment by sandhyadalavi** [comment]
@Ajpantuso could you put some outputs here?

File: README.md [hunk 1]
```diff
@@ -65,12 +65,7 @@ regarding the addition of new customer notifications.
 
 ## Known Issues
 
-### Pager support
-
-The `ocm-cli` offers the ability to configure a pager to send output
```
[the remark this answers is not in the document]
Comment (Ajpantuso) [reply README.md:70]: No, the issue that existed for piping to pagers no longer exists so the issue report has been removed.

**Comment by Ajpantuso** [comment]
> @Ajpantuso could you put some outputs here?

I can't post internal data here, but you can build this PR locally to try:
```
git clone git@github.com:mt-sre/ocm-addons.git
cd ocm-addons
git fetch git@github.com:Ajpantuso/ocm-addons.git table_integration:table_integration
git checkout table_integration
./mage install
```

**Review by sandhyadalavi — APPROVED** [review]
lgtm

**Pull request merged**